src/components/Table/Cells/TableCells.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button, List, ListItem } from '@mui/material';
+import React, { useContext } from 'react';
+import { Store } from '../../../../utils/Store';
+
+export const DeleteServiceCell = ({ row }) => {
```

Comment:
@edfarrera need some help with typescript here. wasn't sure how to define DataGrid cell props here because i was getting typescript errors. maybe you can take a look.